packages/realm/package.json

```diff
@@ -80,22 +79,18 @@
     "test": "wireit",
     "test:types": "tsc --project type-tests/tsconfig.json",
     "lint": "eslint --ext .js,.mjs,.ts .",
+    "prebuild": "tsx ./src/scripts/build/cli.ts",
+    "prebuild-apple": "wireit",
+    "prebuild-apple:simulator": "wireit",
+    "prebuild-android": "wireit",
+    "prebuild-node": "wireit",
     "build:ts": "wireit",
     "build:node": "wireit",
-    "build:node:prebuild": "wireit",
-    "build:node:prebuild:arm": "wireit",
-    "build:node:prebuild:arm64": "wireit",
-    "build:node:prebuild:x64": "wireit",
-    "build:node:prebuild:ia32": "wireit",
```

Comment:
Simply passing `PREBUILD_ARCH` env on CI instead, as I don't expect cross-compiling to be a frequently used script on one of our developers machines.